docker-compose.yml

```diff
@@ -129,5 +129,5 @@ volumes:
   grafana_data:
   influxdb_data:
   shared:
-  op_geth:
```

Comment:
This makes sense if we expect our users to have the data-dir on the host disk. If most of them use docker volumes instead, this will make it harder. I can't judge what the usual case is, but I guess your change is preferable.

Reply:
It's hard to know, we could always support both approaches if there is demand.

At least this approach makes it easier to have a peek inside the datadir which can be beneficial for debugging, which I imagine there will be a fair bit of at network launch time.

Reply:
I agree. I was just unsure enough that I wanted to raise the point.